samples/esb/esb_monitor/sample.yaml

```diff
@@ -0,0 +1,72 @@
+sample:
+  name: ESB Monitor Sample
+tests:
+  sample.esb.monitor:
+    sysbuild: true
+    filter: CONFIG_UART_CONSOLE and CONFIG_SERIAL_SUPPORT_INTERRUPT
+    harness: keyboard
+    integration_platforms:
+      - nrf52dk/nrf52832
+      - nrf52840dk/nrf52840
+      - nrf5340dk/nrf5340/cpunet
+      - nrf54l15dk/nrf54l15/cpuapp
+      - nrf54lm20dk/nrf54lm20a/cpuapp
+    platform_allow:
+      - nrf52dk/nrf52832
+      - nrf52840dk/nrf52840
+      - nrf5340dk/nrf5340/cpunet
+      - nrf54l15dk/nrf54l15/cpuapp
+      - nrf54lm20dk/nrf54lm20a/cpuapp
+    tags:
+      - esb
+      - samples
+      - console
+      - sysbuild
+      - ci_samples_esb
+  sample.esb.monitor.build:
```

Comment:
Finally I'd prefer to merge `sample.esb.monitor.build` and `sample.esb.monitor` test cases to optimize twister time. I am trying to figure out what the filter attribute is used for, but I suspect it may be obsolete.